### PR TITLE
test: improve test runtime 10 fold

### PR DESCRIPTION
**Pull request by coderbyheart**
This significantly reduces the time it takes to run the unit tests.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
   "description": "Converts a JSON document containing the asset_tracker_v2 device and sensor data encoded as LwM2M to the JSON document required by nRF Asset Tracker.",
   "main": "index.js",
   "scripts": {
-    "test": "find ./ -type f -name '*.spec.ts' ! -path '*/node_modules/*' -exec npx tsx {} ';'",
+    "test": "npx tsx --no-warnings --test --test-reporter spec ./src/**/*.spec.ts",
```
Comment (MLopezJ) [inline package.json]: I have the following error when trying to run `npm test`

```
> @nordicsemiconductor/asset-tracker-lwm2m-js@0.0.0-development test
> npx tsx --no-warnings --test --test-reporter spec ./src/**/*.spec.ts


node:internal/worker:211
    this[kHandle] = new WorkerImpl(url,
                    ^
This Environment was initialized without a V8::Inspector
(Use `node --trace-uncaught ...` to show where the exception was thrown)

```

I had that same issue before, that's why I came with `find ./ -type f -name '*.spec.ts' ! -path '*/node_modules/*' -exec npx tsx {} ';'` as the command. 

My node version is`20.0.0` and my npm version is `9.8.1`. 

I tried searching for the error message but I didn't find any useful info about how to solve it. 

Is there any pre-condition needed for this command in order to work properly?

Comment (coderbyheart) [reply package.json]: No, as we can see that the tests in CI pass.

Comment (MLopezJ) [inline package.json]: `src/converter.spec.ts` tests are being ignored by `./src/**/*.spec.ts`. 

I tried with `npx tsx --no-warnings --test --test-reporter spec ./src/*.spec.ts` but the rest of the tests are being ignored then. Also, I tried to use the && command but it didn't work.

**Review by MLopezJ — CHANGES_REQUESTED** [review]
Not all the tests are being executed

File: package.json (hunk 1 above)
Comment (coderbyheart) [reply package.json]: Fixed.

**Review by MLopezJ — APPROVED** [review]

**Pull request merged**